services/tables/src/main/java/com/linkedin/openhouse/tables/dto/mapper/iceberg/PoliciesSpecMapper.java

```diff
@@ -119,24 +120,25 @@ private Replication mapReplicationPolicies(Replication replicationPolicy) {
           replicationPolicy.getConfig().stream()
               .map(
                   replication -> {
-                    if (replication.getInterval() == null || replication.getInterval().isEmpty()) {
-                      return replication
-                          .toBuilder()
-                          .interval(ReplicationInterval.DEFAULT.getInterval())
-                          .cronSchedule(
-                              IntervalToCronConverter.generateCronExpression(
-                                  ReplicationInterval.DEFAULT.getInterval()))
-                          .build();
+                    if (replication.getDestination() == null) {
+                      return replication;
```

Comment:
Does this mean default policy or no replication?

Reply:
this means no replication
I can change this to `replication == null` if it is more clear

Reply:
Yes returning null is better.